scripts/cnode-helper-scripts/deploy-as-systemd.sh

```diff
@@ -26,9 +26,13 @@ if grep -q "^PGPASSFILE=" "${CNODE_HOME}/scripts/dbsync.sh" 2> /dev/null || [[ -
 fi
 
 echo -e "\e[32m~~ Cardano Submit API ~~\e[0m"
-echo "launches the main submitapi.sh script to deploy cardano-submit-api as service"
+echo "Please note that cardano-submit-api will listen on ports on your system to serve the API."
```

Comment:
That message does not really serve a purpose IMO, deploy-as-systemd file should not be describing what a component is, that could be specified in documentation on node-cli.md page